code/_globalvars/lists/maint_loot_tables.dm

```diff
@@ -1,10 +1,12 @@
 GLOBAL_LIST_INIT(maintenance_loot_tier_0, list(
 	list(
 		// Tools
-		/obj/effect/spawner/random/engineering/tools,
+		/obj/effect/spawner/random/engineering/tools = 3,
 
 		// Materials
-		/obj/effect/spawner/random/engineering/materials,
+		/obj/effect/spawner/random/engineering/materials = 3,
+		// Plushies
+		/obj/effect/spawner/random/plushies = 1,
```

Comment:
Also, please mention in the PR description and changelog that you are making specifically tools and materials more rare

Reply:
done.

Reply:
@haydenredacted  I would honestly bump these numbers up a bit, this is an over 10% reduction in those spawns. Could you weight them to 4? We can check the stats once this goes in and make sure everything's sane, and tweak them later.

Reply:
also done.